js/scriptGame.js

```diff
@@ -0,0 +1,130 @@
+let isInventoryOpen = false;
+
+document.getElementsByClassName("inventorySlot slotArrow")[0].onclick = function() {
+    if(!isInventoryOpen){
+        moveInventory("-", 85, 0, 0);
+        isInventoryOpen = true;
+        document.getElementsByClassName("blackoutPanel")[0].classList.remove("hidden")
+    }
+    else if(isInventoryOpen){
+        moveInventory("+", 40, 180, 0.5);
+        isInventoryOpen = false;
+        setTimeout(()=> {
+            document.getElementsByClassName("blackoutPanel")[0].classList.add("hidden");
+        }, 400)
+        
+    }
+}
+
+function moveInventory(direction, startVh, startDeg, startOpacity) {
+    const element = document.getElementsByClassName("playerInventory")[0];
+    const endVh = direction === "+" ? startVh + 45 : startVh - 45;
+    const duration = 300; // ms
+    const startTime = performance.now();
+    const endDeg =  direction === "+" ? startDeg + 180 : startDeg - 180;
+    const endOpacity = direction === "+" ? startOpacity - 0.5 : startOpacity + 0.5;
+
+    function animate(currentTime) {
+        const elapsed = currentTime - startTime;
+        const progress = Math.min(elapsed / duration, 1); // clamp between 0 and 1
+
+        // Interpolate between start and end
+        const currentVh = startVh + (endVh - startVh) * progress;
+        element.style.top = `${currentVh}vh`;
+        const currentDeg = startDeg + (endDeg - startDeg) * progress;
+        document.getElementsByClassName("inventorySlot slotArrow")[0].style.rotate = `${currentDeg}deg`
+        const currentOpacity = startOpacity + (endOpacity - startOpacity) * progress;
+        document.getElementsByClassName("blackoutPanel")[0].style.opacity = currentOpacity;
+
+        if (progress < 1) {
+            requestAnimationFrame(animate);
+        }
+    }
+
+    requestAnimationFrame(animate);
+}
+
+const levelPopUp = document.getElementsByClassName("levelPopUp")[0];
+
+//temporal bit till we get the game sequense going
+
+document.getElementsByClassName("controlButton")[0].onclick = function() {
+    triggerLevelPopUp(100, 0);
+    
+    setTimeout(()=> {
+        levelPopUp.classList.add("hidden");
```

Comment:
Let's have separated function for hidden functionalitty. Change it everywhere where you opearte with "hidden" class. What dou you think about the following:

```suggestion
const HIDDEN_CLASS_NAME = "hidden";

function hideElement(elem) {
    elem.classList.add(HIDDEN_CLASS_NAME);
}

functoin unhideElement(elem) {
    elem.classList.remove(HIDDEN_CLASS_NAME);
}

function isElementHidden(elem) {
   return elem.classList.contains(HIDDEN_CLASS_NAME);
}
```